ROMFS/px4fmu_common/init.d/airframes/13030_generic_vtol_quad_tiltrotor

```diff
@@ -7,18 +7,18 @@
 #
 # @maintainer
 #
-# @output MAIN1 motor 1
-# @output MAIN2 motor 2
-# @output MAIN3 motor 3
-# @output MAIN4 motor 4
-# @output AUX1 Motor tilt front left
-# @output AUX2 Motor tilt front right
-# @output AUX3 Motor tilt rear left
-# @output AUX4 Motor tilt rear right
-# @output AUX5 Aileron left
-# @output AUX6 Aileron right
-# @output AUX7 Elevator
-# @output AUX8 Rudder
+# @output Motor1 motor 1
```

Comment:
The outputs are actually not yet defined (I guess we want to keep that for "Generic" airframes?)

Reply:
Right, removing those